site/_templates/_footer.html

```diff
@@ -1,8 +1,8 @@
     <footer>
         <div class="flex align-center justify-center gap-2 pt-4">
-            <a class="font-medium" href="settings.html">Einstellungen</a>
-            <a class="font-medium" href="imprint.html">Impressum</a>
-            <a class="font-medium" href="data/log.txt">Logs</a>
+            <a class="font-medium" href="settings.html">__Einstellungen__</a>
```

Comment:
You have used `%%path%%` for file includes, I have decided for `__key__` includes for language includes.

I have picked `__` because it is [already associated with translations in the node world](https://github.com/mashpie/i18n-node).